src/components/accountsBottomSheet/view/accountsBottomSheetView.js

```diff
@@ -55,7 +80,7 @@ const AccountsBottomSheet = forwardRef(
           onClose={onClose}
         >
           <FlatList
-            data={accounts}
+            data={[...accounts, ...prevLoggedInUsers]}
```

Comment:
to keep data flow more organised, why not added a section below list instead of mixing different objects? can even add a section labeled `Logged out accounts`